src/main/java/com/google/protobuf/maven/Protoc.java

```diff
@@ -108,8 +109,9 @@
      * @param pythonOutputDirectory a directory into which Python source files will be generated.
      * @param customOutputDirectory a directory into which a custom protoc plugin will generate files.
      * @param descriptorSetFile The directory into which a descriptor set will be generated;
-     * if {@code null}, no descriptor set will be written
+* if {@code null}, no descriptor set will be written
      * @param includeImportsInDescriptorSet If {@code true}, dependencies will be included in the descriptor set.
+     * @param includeSourceInfoInDescriptorSet
```

Comment:
Can you please add javadoc for the `includeSourceInfoInDescriptorSet` parameter (otherwise Java8 doclint will break the build), and revert the javadoc changes in the lines above, which seem to be a result of "javadoc auto-format gone wild".

Otherwise all good, thank you for the contribution!